src/ast/dml.rs

```diff
@@ -124,6 +124,7 @@ pub struct CreateTable {
     pub default_charset: Option<String>,
     pub collation: Option<String>,
     pub on_commit: Option<OnCommit>,
+    pub with_order: Option<Vec<Vec<OrderByExpr>>>,
```

Comment:
I think we don't need `Option` wrap. Doesn't being empty of the first vector and `None` have the same meaning?